modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/OutputInfoM.scala

```diff
@@ -0,0 +1,20 @@
+package org.ergoplatform.explorer.http.api.v1.models
+
+import org.ergoplatform.explorer.http.api.models.Items
+import derevo.circe.{decoder, encoder}
+import derevo.derive
+import sttp.tapir.{Schema, Validator}
+
+@derive(encoder, decoder)
+final case class OutputInfoM(filteredBoxes: Items[OutputInfo], mempoolBoxes: Items[UOutputInfo])
```

Comment:
Why so weird response format is chosen? I think it's better to return union of all boxes (mempool+non-mempool) where each entry is wrapped into some kind of wrapper carrying `confirmed` (`confirmed=false` for mempool boxes) flag.

Reply:
I have done this;
used mainChain=false for Mempool boxes